### PR TITLE
Adds Prettier apex

### PR DESCRIPTION
**Pull request by codefriar**
Also formats the two apex classes.

### CODE REVIEW

File: redwoods-insurance-salesforce/package.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
         "eslint": "^5.15.1",
         "husky": "^3.0.5",
         "lint-staged": "^9.3.0",
-        "prettier": "^1.18.2",
+        "prettier": "1.18.2",
```
Comment (muenzpraeger) [inline redwoods-insurance-salesforce/package.json]: We should allow version upgrades by prepending `^` to the version number.

Comment (muenzpraeger) [reply redwoods-insurance-salesforce/package.json]: Also prettier related scripts in `package.json` don't have file extensions added for Apex classes and triggers.

Comment (codefriar) [reply redwoods-insurance-salesforce/package.json]: Fair, adjusting ;)

**Comment by codefriar** [comment]
@muenzpraeger  - I've added the ^ and added 'cls,trigger' to the file extensions list in 3 places. Manual testing with npm run prettier include the apex classes. No Triggers are (currently, or planned) included, so trigger formatting has not been tested.

**Pull request merged**